src/clj_rn/shell.clj

```diff
@@ -1,6 +1,16 @@
-(ns clj-rn.shell)
+(ns clj-rn.shell
+  (:require [clojure.string :as str]
+            [clojure.java.io :as io]))
+(import 'java.lang.Runtime)
```

Comment:
Is this expected to be out of the ns declaration?

Reply:
No reason. I will change it.